### DISCUSSION TITLE
Does not work on Win11

### DISCUSSION THREAD

**Discussion by denivic** [flexits/HotCornersWin#20] [Q&A]
I have tried many combinations of settings. I'm running Win11 Pro N. Any ideas?

**Comment by flexits** [comment]
Hello. I'm not using Win 11 currently, so I won't be able to check right now. However, the application was tested by several users and was reported to be fine. 
Please provide more details. What settings are you talking about exactly? What is the state of the application: disabled (the tray icon is gray) or enabled (the icon is blue)? Don't you by chance use GeForce Experience (issue #15)?

**Reply by denivic** [reply]
I was wondering how I seemed to be the only one with issues. I'm using an AMD GPU and the tray icon is blue.

I'm currently unable to provide more details (Im on my phone), but I will comment later on. Some things to note:

- I set my taskbar to hide automatically. I don't know if that might interfere somehow.
- My Win11 is modified in numerous ways, but none that should have any effect on this program. However I'll try a fresh install on VM later.

Edit: The above will have to wait until tomorrow.
